### PR TITLE
remove viz from main branch

### PR DESCRIPTION
**Pull request by boneskull**
- **chore(viz): remove**
- **chore: update lockfile & release-please**
- **docs(README): update README with viz info**

`lavamoat-viz` was extracted via `git subtree split --prefix packages/viz -b lavamoat-viz` and pushed to the [lavamoat-viz branch](https://github.com/LavaMoat/LavaMoat/tree/lavamoat-viz).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,26 +67,28 @@ LavaMoat modifies the bundle's runtime to enforce the configured constraints.
...
 - [LavaMoat Browserify][LavamoatBrowserify]
 
-### Lavamoat Viz
+### lavamoat-viz
 
-[lavamoatViz]: https://lavamoat.github.io/LavaMoat/
+> [!WARNING] `lavamoat-viz` is currently unmaintained; we have tentative plans to resume development in the future.
 
-The [lavamoat viz][lavamoatViz] is a tool to visualize an application's dependency graph and assess package dangerousness.
+[lavamoatViz]: https://github.com/LavaMoat/LavaMoat/tree/lavamoat-viz
```
Comment (naugtur) [inline README.md:74]: (could) link to the branch where it's preserved

Comment (boneskull) [reply README.md:74]: that's what this is dogg

**Comment by boneskull** [comment]
* **#1079** <a href="https://app.graphite.dev/github/pr/LavaMoat/LavaMoat/1079?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @boneskull and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by boneskull** [comment]
I aim to cut a release if/when this is merged

File: README.md (hunk 1 above)
Comment (naugtur) [reply README.md:74]: brainfart

**Review by naugtur — APPROVED** [review]

**Pull request merged**